apps/client/src/features/app-settings/panel/project-panel/ProjectPanel.module.scss

```diff
@@ -6,5 +6,42 @@
 }
 
 .actionButton {
+  flex: 1;
   text-align: right;
 }
+
+.form {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.formInput {
+  flex: 2;
+}
+
+.actionButtons {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  margin-left: 1rem;
+}
+
+.error {
+  color: $red-500;
+}
+
+.createContainer {
+  padding: 0.5rem 0;
+}
+
+.saveButton {
+  text-transform: capitalize;
+}
```

Comment:
todo: this style doesnt seem to apply or be necessary, i suggest we remove it

Reply:
Without this the buttons will be lower case, like `duplicate` or `rename`.

I could change the action names to be `Duplicate` | `Rename`, but I usually prefer to use all lowercase for action names, but we can change that

Reply:
you are correct. sorry for the confusion